src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,151 @@
+package duke.main;
+
+import duke.exception.CannotSaveFileException;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+import duke.ui.DialogBox;
+import duke.ui.Ui;
+import javafx.application.Application;
```

Comment:
I think you should leave a line between duke imports and javafx imports but I'm not too sure also.

Reply:
I've included the newline anyway, as it looks neater to me too. Thanks!